compat/xpipe.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef COMPAT_XPIPE_H
```

Comment:
Please use `SQUID_<DIR>_<NAME>_H` pattern for new guards to reduce clash risk and to ease code maintenance.

Reply:
Right. source-maintenance fixed it for me.